Interpreter report (#73)
chhoumann/p4: Report/sections/3_implementation/interpreter.tex

\section{The Game}
Following the code generation phase is the final result of the code compilation - the game itself. 

> This is built using the Unity engine and interprets the intermediate models in order to build the world defind by the user in the \dazel{} source code. As can be seen in \figref{Dazel_Title}, the user is presented with a few options when launching the application.

Reviewer: ```suggestion
The game is built using the Unity engine. It interprets the intermediate models in order to build the world defind by the user in the \dazel{} source code. As can be seen in \figref{Dazel_Title}, the user is presented with a few options when launching the application.
```

Reviewer: Personal opinion: one sentence, one idea. This is a nit - you can disregard if you don't agree.

\fig{0.8}{Dazel_Title}{The title screen of Dazel. Grass graphics made by Bonsaiheldin\cite{bonsaiheldin_grass_2016}.}
 
The "Play" button runs the \dazel{} compiler and, if compilation succeeded, runs the interpreter that builds the game from the intermediate models. 

> The "Enable logging" toggle enables or disables the logging of messages, warnings and errors in the console GUI described in \fxfatal{missing reference to the section about error handling}.

Reviewer: ```suggestion
The "Enable logging" checkbox toggles the logging of messages, warnings and errors in the console GUI described in \fxfatal{missing reference to the section about error handling}.
```

Reviewer: Nit lmao
